packages/monitoring/src/v3/index.js

```diff
@@ -31,7 +32,11 @@ function v3(options) {
   return result;
 }
 
+v3.GAPIC_VERSION = '0.0.5';
 v3.SERVICE_ADDRESS = groupServiceClient.SERVICE_ADDRESS;
-v3.ALL_SCOPES = groupServiceClient.ALL_SCOPES;
+v3.ALL_SCOPES = union(
```

Comment:
The `lodash.union` dependency isn't needed-- these `ALL_SCOPES` values are identical.